components/rowListScene.brs

```diff
@@ -1,12 +1,21 @@
 sub init()
     m.global.addField("http", "node", FALSE)
     m.global.addField("ratio", "float", FALSE)
+    m.global.addField("mainScreen", "node", FALSE)
+    m.global.mainScreen = m.top
     m.global.http = createObject("roSGNode", "httpTask")
     m.global.http.control = "RUN"
     m.global.ratio = 1
```

Comment:
It is better that you use regular fields instead of `m.global`. The reason behind this is performance and style. `m.global` will require a rendezvous if accessed from several Tasks (threads.)

Reply:
Got it

Reply:
Is it possible to remove all globals except for `m.global.ratio`?

Reply:
Done